chainer/functions/noise/simplified_dropconnect.py

```diff
@@ -127,6 +133,8 @@ def simplified_dropconnect(x, W, b=None, ratio=.5, train=True, mask=None):
         b (~chainer.Variable): Bias variable (optional) of shape ``(M,)``.
         ratio (float):
             Dropconnect ratio.
+        batchwise_mask (bool):
+            If ``True``, dropped value depends on each sample in mini-batch.
```

Comment:
It seems to me that `value` indicates the activation, or the output of the function. So, how about changing it with "dropped connections depend on ..."?